web/src/main/java/org/phoenixctms/ctsms/web/jersey/resource/PATCH.java

```diff
@@ -1,14 +0,0 @@
-package org.phoenixctms.ctsms.web.jersey.resource;
-
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
-
-import javax.ws.rs.HttpMethod;
-
-@Target({ ElementType.METHOD })
-@Retention(RetentionPolicy.RUNTIME)
-@HttpMethod("PATCH")
-public @interface PATCH {
-}
```

Comment:
well the plan is to introduce PATCH support at some point, so why does this has to be removed?

Reply:
Jersey 2 includes PATCH. An upgrade would be more efficient than maintaining unused functionality in the current code base.